src/Data/Bool/Properties.agda

```diff
@@ -660,6 +647,56 @@ not-¬ {false} refl ()
 ¬-not {false} {true}  _   = refl
 ¬-not {false} {false} x≢y = ⊥-elim (x≢y refl)
 
+------------------------------------------------------------------------
+-- Properties of _xor_
+
+xor-is-ok : ∀ x y → x xor y ≡ (x ∨ y) ∧ not (x ∧ y)
+xor-is-ok true  y = refl
+xor-is-ok false y = sym (∧-identityʳ _)
+
+xor-same : ∀ x → x xor x ≡ false
+xor-same false = refl
+xor-same true  = refl
+
+xor-false : ∀ x → false xor x ≡ x
```

Comment:
In the same way we do for `and` and `or` can we use the pre-defined types in `Algebra.Definitions`  e.g. identity, comm etc, and change the names to match?